app/controllers/complaints_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ComplaintsController < ApplicationController
   def index
-    @complaints = FakeData::Complaints.generate_response[:data]
+    @complaints = FakeData::ApiResponse.generate_hses_response[:data]
```

Comment:
[nitpick] this is likely too picky for something that is designed to be thrown away, but I feel like this should reference the type of hses response it is. `generate_hses_issues_response` maybe?

Reply:
[thought] should we keep the fake data around longer to support pa11y scans?

Reply:
i think we should keep it around to support scans and general tests. 

[thought] if we do that should we move FakeData to `/models` where it can be a more proper class?

Reply:
> if we do that should we move FakeData to `/models` where it can be a more proper class?

I think so, yes

Reply:
hooooray